frontend/src/components/HomePage/index.jsx

```diff
@@ -1,74 +1,97 @@
+    "mitigation", "sustainability"
+  ],
+  "Transportation": [
+    "transit", "vehicles", "mobility", "infrastructure", "equity", "bikes", "cities", "mass", "rail",
+    "sharing", "carpooling", "shared", "smart", "public", "bus", "accessibility", "oriented", "bicycle",
+    "urban", "expansion", "sustainable", "micro", "zero", "congestion"
+  ],
+  "Fair Housing": [
+    "housing", "equity", "justice", "inclusive", "affordability", "security", "control", "assistance",
+    "rights", "discrimination", "opportunities", "community", "development", "opportunities", "solutions",
+    "regulation", "advocacy", "planning", "policies", "initiatives", "cooperatives", "availability", "rent"
+  ],
+  "Social Justice": [
+    "equality", "justice", "equity", "society", "rights", "reform", "opportunity", "diversity",
+    "access", "welfare", "dignity", "communities", "opportunities", "harmony", "reconciliation",
+    "policies", "access", "inclusion", "democracy", "development"
+  ],
+  "Education": [
+    "education", "school", "teacher", "student", "curriculum", "equity", "access", "reform",
+    "training", "learning", "childhood", "higher", "STEM", "technology", "parent", "special",
+    "resources", "policy", "infrastructure", "achievement", "funding", "retention", "readiness"
+  ],
+  "Government": [
+    "legislature", "executive", "judiciary", "administration", "policy",
+    "governance", "legislation", "regulation", "budget", "oversight",
+    "authority", "accountability", "federal", "state", "local", "municipal",
+    "council", "mayor", "governor", "president", "congress", "senate", "house",
+    "law", "regulation", "committee", "commission", "agency", "department",
+    "bureau", "office", "public", "official", "civil", "servant", "leadership",
+    "rule", "decision", "policy", "legislation", "governmental", "civic",
+    "constitution", "statute", "ordinance", "charter", "administration",
+    "judiciary"
+  ],
+  "Healthcare": [
+    "healthcare", "health", "medical", "hospital", "doctor", "patient",
+    "nurse", "clinic", "medicine", "insurance", "provider", "treatment",
+    "wellness", "pharmacy", "coverage", "vaccine", "disease", "emergency",
+    "public health", "health system", "pandemic", "care", "mental health"
+  ]
+};
+
+// Function to get up to 3 categories for a bill based on keyword count
+const getCategories = (summary) => {
+  const matchingCategories = Object.entries(categories)
+    .filter(([category, keywords]) => keywords.some(keyword => summary.toLowerCase().includes(keyword)))
+    .map(([category]) => category);
+  return matchingCategories.slice(0, 3);
+};
+
+
 const HomePage = () => {
   const [bills, setBills] = useState([]);
-  const [categoryMappings, setCategoryMappings] = useState({});
-  const [categories, setCategories] = useState({});
 
   useEffect(() => {
-
-    // Correctly handle double-mount in dev/StrictMode
-    // https://stackoverflow.com/a/72238236
```

Comment:
Why remove this comment?

Reply:
Odd - it still exists in the version I have locally. No reason it should be removed